ejb3/src/main/java/org/jboss/as/ejb3/component/EJBValidationConfigurator.java

```diff
@@ -21,7 +21,7 @@
 
 /**
  *
- * Configurator that validates than an Jakarta Enterprise Beans class does not validate the Jakarta Enterprise Beans specification
+ * Configurator that validates than a Jakarta Enterprise Beans class does not validate the Jakarta Enterprise Beans specification
```

Comment:
s/than/that

also, the 2nd "validate" should be "violate"

Reply:
Fixed both